src/compiler/diagnosticMessages.json

```diff
@@ -880,7 +880,7 @@
         "category": "Error",
         "code": 1308
     },
-    "'=' can only be used in an object literal property inside a destructuring assignment.": {
+    "'=' can only be used in an object literal property inside a destructuring assignment. Did you mean to use a ':'?": {

```

Comment:
Okay, last thing I want to nit.

Part of me wants to rephrase this.

```
When following object literal property names, '=' implies a destructuring assignment. Did you mean to use a ':'? 
```

Going even further, I'm tempted to put the most actionable information first, even though most of our error messages give the problem, then the "Did you mean?" portion.

But the information at the beginning is so long that I want to front-load the suggestion:

```
Did you mean to use a ':'? When following object literal property names, '=' implies a destructuring assignment.
```

@sandersn

Reply:
Yes, that is way better! (The latter, tempting one especially.)